### ISSUE TITLE
use taskwarrior column-format options as a model for displaying description/ annotation

### ISSUE THREAD

**Issue by linuxcaffe** [blakesweeney/unite-taskwarrior#46]
How should annotations be displayed in a utw tasklist?

well taskwarrior its self has several option for displating descriptions, shown in the 'task columns' command;

<pre>
description    string   combined*         Move your clothes down on to the lower peg
                                            09/06 15:37 Immediately before your lunch
                                            09/06 15:37 If you are playing in the match this
                                          afternoon
                                            09/06 15:37 Before you write your letter home
                                            09/06 15:37 If you're not getting your hair cut
                        desc              Move your clothes down on to the lower peg
                        oneline           Move your clothes down on to the lower peg 09/06 15:37
                                          Immediately before your lunch 09/06 15:37 If you are
                                          playing in the match this afternoon 09/06 15:37 Before
                                          you write your letter home 09/06 15:37 If you're not
                                          getting your hair cut
                        truncated         Move your clothes do...
                        count             Move your clothes down on to the lower peg [4]
                        truncated_count   Move your clothes do... [4]
(*=default)
</pre>

and these ideas could (should?) be options for utw. I'm not sure how this would be implemented or controlled, but I do know these are well-thought out and familiar in concept, for taskwarrior users.

This is related to #44

**Comment by linuxcaffe** [comment]
vim-taskwarrior has the ability to cycle through available column formats, and it's mapped to "H" (to cycle left) and "L" (to cycle to the right, ironically), The "right and left" are arbitrary directions through the available options,

This code might be helpful, and something like this behavior; "format cycling" through available description(+annotation) column formats, would be ideal

**Comment by blakesweeney** [comment]
Changing the annotation thing seems pretty possible. Right now there are some more pressing issues so this isn't high priority.

**Comment by linuxcaffe** [comment]
agreed, just suggesting 2 models 1) tw column formats, and 2) tim-tw's format cycling. I titally agree that this isn't pressing.

idea: set up some labels, v1.0 and v1.2 and backlog and wiki, and many of my mad ideas can be properly backburnered ;-)

**Comment by blakesweeney** [comment]
Yeah, I need to go through and organize all the issues.

**Comment by linuxcaffe** [comment]
err... yeah, sorry about the idea-storm! I think the worst of it has passed :)

**Comment by blakesweeney** [comment]
This may not be too hard. I'm marking as 1.1, but may get to it for 1.0.

**Comment by linuxcaffe** [comment]
jah, cool!
